circle.yml

```diff
@@ -322,8 +327,7 @@ jobs:
       JOBS: 4
       BUILDTYPE: RelWithDebInfo
       WITH_EGL: 1
-      PACKAGE_JSON_VERSION: $(node -e "console.log(require('./package.json').version)")
-      PUBLISH: $([[ "${CIRCLE_BRANCH}" == "node-v${PACKAGE_JSON_VERSION}" ]] && echo true)
```

Comment:
Setting env variables seems to not work which is why I added this logic [`platform/node/scripts/after_success.sh`](https://github.com/mapbox/mapbox-gl-native/pull/9610/files#diff-7f6ebb693582f9045c60fa9dcf7a46efR6)